backend/clickhouse/logs.go

```diff
@@ -86,11 +87,48 @@ func (client *Client) ReadLogs(ctx context.Context, projectID int, params modelI
 
 		logLines = append(logLines, &modelInputs.LogLine{
 			Timestamp:     Timestamp,
-			SeverityText:  SeverityText,
+			SeverityText:  makeSeverityText(SeverityText),
 			Body:          Body,
 			LogAttributes: gqlLogAttributes,
 		})
 	}
 	rows.Close()
 	return logLines, rows.Err()
 }
+
+func makeSeverityText(severityText string) modelInputs.SeverityText {
```

Comment:
Ensures that every log severity text is present and fits into one of the following [types](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#severity-fields).

The client sdk sends in the types as things like `log`, `console.error`, etc: https://github.com/highlight/highlight/blob/ae909c5aa610b1f1dac9223adc423a77941e26bc/sdk/client/src/types/shared-types.tsx#L6-L12

But these have no meaning in OTEL and we want the log viewing to be consistent on the frontend.